charts/prometheus-elasticsearch-exporter/Chart.yaml

```diff
@@ -3,7 +3,7 @@ description: Elasticsearch stats exporter for Prometheus
 name: prometheus-elasticsearch-exporter
 version: 5.1.1
```

Comment:
Thank you for your contribution. Please, see comments below.

```suggestion
version: 5.2.0
```

As the application itself is not being changed, it is `version` that needs a bump.